docs/version-history/development-release-series-89.rst

```diff
@@ -1979,7 +1979,7 @@ Release Notes:
 Known Issues:
 
 This release may require configuration changes to work as before. During
-this release series, we are making changes to make it easier to deploy
+the 8.9 development, we are making changes to make it easier to deploy
```

Comment:
Should be either "the development of 8.9", "the 8.9 development series", or "As we develop HTCondor..."